specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2020-07-30-preview/HybridCompute.json

```diff
@@ -0,0 +1,1211 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hybrid machine."
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on the operation.",
+            "enum": [
+              "instanceView"
+            ],
+            "x-ms-enum": {
+              "name": "InstanceViewTypes",
+              "modelAsString": true
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Machine"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Machine": {
+            "$ref": "./examples/Machines_Get.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HybridCompute/machines": {
+      "get": {
+        "tags": [
+          "machines"
+        ],
+        "operationId": "Machines_ListByResourceGroup",
+        "description": "Lists all the hybrid machines in the specified resource group. Use the nextLink property in the response to get the next page of hybrid machines.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MachineListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Machines by resource group": {
+            "$ref": "./examples/Machines_ListByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.HybridCompute/machines": {
```

Comment:
What response will users get when they call /reconnect in an older api-version?

Reply:
The response from ARM of reconnect is the same as connect. For example,
{
  "parameters": {
    "subscriptionId": "{subscriptionId}",
    "resourceGroupName": "myResourceGroup",
    "name": "myMachine",
    "api-version": "2019-12-12",
    "parameters": {
      "properties": {
        "vmId": "b7a098cc-b0b8-46e8-a205-62f301a62a8f",
        "clientPublicKey": "string"
      }
    }
  },
  "responses": {
    "200": {
      "body": {
        "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.HybridCompute/machines/myMachine",
        "name": "myMachine",
        "location": "eastus2euap",
        "tags": null,
        "identity": {
          "type": "SystemAssigned",
          "principalId": "string",
          "tenantId": "string"
        },
        "type": "Microsoft.HybridCompute/machines",
        "properties": {
          "provisioningState": "Succeeded",
          "agentVersion": null,
          "status": null,
          "lastStatusChange": null,
          "errorDetails": null,
          "displayName": null,
          "machineFqdn": null,
          "osName": null,
          "osVersion": null,
          "osProfile": {
            "computerName": null
          },
          "vmId": "b7a098cc-b0b8-46e8-a205-62f301a62a8f",
          "locationData": {
            "name": "Redmond"
          },

        }
      }
    }
  }
}

Reply:
no error with reconnect if using old api for now. we will either update our RP code to respond "Deprecated".  Or simply remove reconnect from our RP roledefinition.
However we have removed the reconnect from our agent code already which is a downloadable package to be run on machines in the August release.